check_process

```diff
@@ -13,10 +13,8 @@
 		setup_public=1
 		upgrade=1
 		backup_restore=1
-		multi_instance=1
+		multi_instance=0
```

Comment:
Maybe, you should also disable `"multi_instance": true,` in manifest.json, but not sure of that

Reply:
The bug happens only in the CI because we use `domain.tld` for the second instance.
Can I set a domain for the `multi_instance` test ? Like `subdomain2.domain.tld` ?

Reply:
> 
> 
> The bug happens only in the CI because we use `domain.tld` for the second instance (and it's blocked by some lib in glichsoc).
> Can I set a domain for the `multi_instance` test ? Like `subdomain2.domain.tld` ?

I don't know :/